MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -612,6 +619,93 @@ void EnggDiffractionViewQtGUI::plotReplacingWindow(const std::string &wsName) {
   m_presenter->notify(IEnggDiffractionPresenter::LogMsg);
 }
 
+void EnggDiffractionViewQtGUI::plotVanCurvesCalibOutput() {
+  std::string pyCode =
+      "van_curves_ws = workspace(\"engggui_vanadium_curves_ws\")\n"
+      "plot(van_curves_ws, [0, 1, 2])";
+
+  std::string status =
+      runPythonCode(QString::fromStdString(pyCode), false).toStdString();
+
+  m_logMsgs.push_back(
+      "Plotted output calibration vanadium curves, with status string " +
+      status);
+  m_presenter->notify(IEnggDiffractionPresenter::LogMsg);
+}
+
+void EnggDiffractionViewQtGUI::plotDifcZeroCalibOutput(
+    std::vector<double> &difc, std::vector<double> &tzero,
+    std::string &specNo) {
+
+  size_t bank1 = size_t(0);
+  size_t bank2 = size_t(1);
+  std::string pyRange;
+
+  // sets the range to plot appropriate graph for the particular bank
+  if (specNo == "North") {
+    pyRange = "1, 2";
+  } else if (specNo == "South") {
+    pyRange = "2, 3";
+    bank2 = size_t(0);
+  } else {
+    pyRange = "1, 3";
+  }
+
+  std::string pyCode =
+      "for i in range(" + pyRange +
+      "):\n"
+      " bank_ws = workspace(\"engggui_calibration_bank_\" + str(i))\n"
+
+      " xVal = []\n"
+      " yVal = []\n"
+      " y2Val = []\n"
+
+      " for irow in range(0, bank_ws.rowCount()):\n"
+      "  xVal.append(bank_ws.cell(irow, 0))\n"
+      "  yVal.append(bank_ws.cell(irow, 5))\n"
+
+      "  if (i == 1):\n"
+      "   difc=" +
+      boost::lexical_cast<std::string>(difc[bank1]) + "\n" + "   tzero=" +
+      boost::lexical_cast<std::string>(tzero[bank1]) + "\n" + "  else:\n"
+
+                                                              "   difc=" +
+      boost::lexical_cast<std::string>(difc[bank2]) + "\n" + "   tzero=" +
+      boost::lexical_cast<std::string>(tzero[bank2]) + "\n" +
+
+      "  y2Val.append(xVal[irow] * difc + tzero)\n"
+
+      " ws1 = CreateWorkspace(DataX=xVal, DataY=yVal, UnitX=\"Expected "
+      "Peaks "
+      " Centre(dSpacing, A)\", YUnitLabel = \"Fitted Peaks Centre(TOF, "
+      "us)\")\n"
+      " ws2 = CreateWorkspace(DataX=xVal, DataY=y2Val)\n"
+      " output_ws = \"engggui_difc_zero_peaks_bank_\" + str(i)\n"
+      " AppendSpectra(ws1, ws2, OutputWorkspace=output_ws)\n"
+
+      " DeleteWorkspace(ws1)\n"
+      " DeleteWorkspace(ws2)\n"
+
+      " DifcZeroWs = \"engggui_difc_zero_peaks_bank_\" + str(i)\n"
+
+      " DifcZeroWs = workspace(DifcZeroWs)\n"
+      " DifcZeroWs = plotSpectrum(DifcZeroWs, [0, 1]).activeLayer()\n"
+
+      " DifcZeroWs.setTitle(\"Engg Gui Difc Zero Peaks Bank \" + "
+      "str(i))\n"
+      " DifcZeroWs.setAxisTitle(Layer.Bottom, \"Expected Peaks "
+      "Centre(dSpacing, "
+      " A)\")\n"
+      " DifcZeroWs.setCurveLineStyle(0, QtCore.Qt.DotLine)\n";
+
```

Comment:
I noticed that now the interfaces creates these workspaces `engggui_difc_zero_peaks_bank_i` . This should be mentioned in the documentation, explaining that their content is what is shown in the plots (peaks used for the difc/tzero fitting per bank).